vispy/gloo/texture.py

```diff
@@ -176,7 +195,16 @@ def resize(self, shape, format=None):
             The format of the texture: 'luminance', 'alpha',
             'luminance_alpha', 'rgb', or 'rgba'. If not given the format
             is chosen automatically based on the number of channels.
-            When the data has one channel, 'luminance' is assumed.
+            When the data has one channel, 'luminance' is assumed.  
+        internalformat : str | enum
```

Comment:
`str | enum | None` is more complete, then below you can say "if None" instead of "if not given"